static/css/applause-button.css

```diff
@@ -0,0 +1,212 @@
+applause-button {
```

Comment:
Create a `applause-button` folder inside static. Please the CSS and JS file in it.

Reply:
After adding the CSS file in the folder, you will have to add link to it here

https://github.com/Pelican-Elegant/elegant/blob/master/gulpfile.babel.js#L78

Reply:
Done